Shopify/View/HomeScreens/HomeViewController.swift

```diff
@@ -95,7 +120,7 @@ extension HomeViewController : UICollectionViewDataSource
             cell.layer.borderWidth   = 3.0
             cell.layer.cornerRadius  = 25.0
             cell.configImg(name: "tmpBrand")
```

Comment:
we have to have a  discussion  about create a  class for UI Functions and Call it whenever and wherever we need them to make out View Controllers more clear and readable